packages/desktop-client/src/components/reports/CustomTick.tsx

```diff
@@ -0,0 +1,9 @@
+const CustomTick = (value: string, test: boolean) => {
+  if (!test) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the naming of this variable should be improved
```suggestion
  if (!isPrivacyModeEnabled) {
```